tests/everflow/test_everflow_ipv6.py

```diff
@@ -0,0 +1,592 @@
+                      "Incomplete packet, expected at least {} header bytes".format(self.OUTER_HEADER_SIZE))
+
+        inner_frame = encapsulated_packet[-payload_size:]
+        return packet.Ether(inner_frame)
+
+    def _base_tcp_packet(self,
+                         ptfadapter,
+                         setup,
+                         src_ip=DEFAULT_SRC_IP,
+                         dst_ip=DEFAULT_DST_IP,
+                         next_header=None,
+                         dscp=None,
+                         sport=2020,
+                         dport=8080,
+                         flags=0x10):
+        pkt = testutils.simple_tcpv6_packet(
+            eth_src=ptfadapter.dataplane.get_mac(0, 0),
+            eth_dst=setup["router_mac"],
+            ipv6_src=src_ip,
+            ipv6_dst=dst_ip,
+            ipv6_dscp=dscp,
+            ipv6_hlim=64,
+            tcp_sport=sport,
+            tcp_dport=dport,
+            tcp_flags=flags,
+        )
+
+        if next_header:
+            pkt["IPv6"].nh = next_header
+
+        return pkt
+
+    def _base_udp_packet(self,
+                         ptfadapter,
+                         setup,
+                         src_ip=DEFAULT_SRC_IP,
+                         dst_ip=DEFAULT_DST_IP,
+                         next_header=None,
+                         dscp=None,
+                         sport=2020,
+                         dport=8080):
+        pkt = testutils.simple_udpv6_packet(
+            eth_src=ptfadapter.dataplane.get_mac(0, 0),
+            eth_dst=setup["router_mac"],
+            ipv6_src=src_ip,
+            ipv6_dst=dst_ip,
+            ipv6_dscp=dscp,
+            ipv6_hlim=64,
+            udp_sport=sport,
+            udp_dport=dport,
+        )
+
+        if next_header:
+            pkt["IPv6"].nh = next_header
+
+        return pkt
+
+
+class TestIngressEverflowIPv6(EverflowIPv6Tests):
+    """Parameters for Ingress Everflow IPv6 testing. (Ingress ACLs/Ingress Mirror)"""
+    def acl_stage(self):
+        return "ingress"
+
+    def mirror_type(self):
+        return "ingress"
+
+    @pytest.fixture(scope='class',  autouse=True)
+    def setup_acl_table(self, duthost, setup_info, setup_mirror_session):
+        duthost.command("rm -rf everflow_v6_tests")
+        duthost.command("mkdir -p everflow_v6_tests")
+
+        table_name = self._get_table_name(duthost)
+        temporary_table = False
+
+        if not table_name:
+            table_name = "EVERFLOWV6"
+            temporary_table = True
+            duthost.command("config acl add table {} MIRRORV6".format(table_name))
+
+        duthost.host.options["variable_manager"].extra_vars.update({"acl_table_name": table_name})
+        duthost.template(src="everflow/templates/acl_rule_v6.json.j2", dest="everflow_v6_tests/acl_rule_v6.json")
+        duthost.command("acl-loader update full everflow_v6_tests/acl_rule_v6.json --session_name={}".format(setup_mirror_session["session_name"]))
+
+        yield
+
```

Comment:
We don't have mirror with  policer use case for Ipv6 Traffic ?

Reply:
We typically use a policer with the MIRROR_DSCP table.